docs/pages/0115-command-reference.md

```diff
@@ -0,0 +1,226 @@
+title: Command Reference
+
+---
+
+## Command Reference
```

Comment:
The first 4 lines might be redundant?
```suggestion
## Command Reference
```
![Screenshot 2024-05-08 at 16 49 20](https://github.com/degica/kaiser/assets/1732016/cc15fc6a-60a2-4116-8955-ca1bafabac08)

Reply:
those are called the front matter for the html generator to use

Reply:
Oh really, interesting 😮 
Thank you for letting me know.